digital-anonymity.md

```diff
@@ -0,0 +1,161 @@
+| :---- | :---- |
+| Title | Digital Anonymity - Is it Achievable? |
+| Headline  | Looking at the possibility for Digital Anonymity and if its achievable. |
+| Cypherpunktimes | link |
+| Publish Date | xx Mar 2024 |
+
+| Social Media | Link |
+| :---- | :---- |
+| Twitter | twitter link |
+| Twitter | thread link… |
+| Twitter | poll link… |
+| Reddit  | link… |
+| Bison Relay | News feed title |
+| Other platforms | Instagram and TikTok |
+
+Thesis statement - The piece aims to look at digital anonymity. With increasing surveillance is it possible to achieve.
+
+Pre-discussion, research and questions
+
+Tagline to use : Anonymity Online - Is this Possible nowadays?
+
+
+# Digital Anonymity - Is it Achievable?
+
+
+Todays world has ever increasing surveillance. Preserving digital anonymity must be a primary concern.
+
+This article will look at the importance of having anonymity online and the challenges and obstacles that prevent this. We will look at the role of cryptocurrency and other communication tools that try to protect and preserve anonymity.
+
+
+## Challenges We Face:
+
+There are a large number of challenges to overcome to maintain anonymity online. Constantly activites are being monitored or tracked, data is being exploited or hacked. Whether the challenge comes from government surveillance, corporations collecting data, cyber attacks, people face a huge threat daily to their privacy.
+
+Below is a list of constant threats to anonymity:
+
+1. Government Surveillance:
+
+There has been increased investment in surveillance technology and this often comes with the tagline 'Its in the interest of national security'. The seemingly open collaboration between governments and tech companies is worrying in so far as retention of data, spying and how this can be seen as an invasion of privacy. Laws have been getting up to speed with combatting this but have they gone far enough. In Europe we have General Data Protection Regulation (GDPR) which offers some protection. A direct quote and meaning of this is:
+
+'GDPR stands for General Data Protection Legislation. It is a European Union (EU) law that came into effect on 25th May 2018. GDPR governs the way in which we can use, process, and store personal data (information about an identifiable, living person).'
+
+Retaining anonymity and protecting oneself when governments continue to implement more and more tracking online is a serious threat to privacy. You could imagine a scenario where you were tracked online buying certain illicit substances, whether you actually made the purchase is unclear or received the substances but maybe this will be stored in a government database that person X from IP address X is linked with brining illicit substances into the country. You can also see the reason why this may be seen in the national interest also as you want to know that the government has the ability to capture anyone that may be brining something in to the country to harm its citizens.
```

Comment:
"may be brining something" should be "may be bringing something"

Before I continue my review, could you run this text through Grammarly to check and correct spelling and grammar issues. I've spotted a few but adding multiple comments for spelling might get a little annoying for you to correct individually.

Reply:
I've given the piece a full read. Generally it flows very well and includes good point about the data being collected and how it is used. I also liked the examples given for why sharing data is a problem.

Once the grammar and spelling issues are taken care of I'll give it a more detailed read.

Reply:
Grammar and spelling resolved